src/convert.jl

```diff
@@ -46,6 +82,110 @@ function convert(::Type{HPolytope}, P::AbstractHPolygon)
     return HPolytope(P.constraints)
 end
 
+"""
+    convert(::Type{VPolytope}, P::VPolygon)
+
+Convert from polygon in V-representation to polytope in V-representation.
+
+### Input
+
+- `type` -- target type
+- `P`    -- source polygon
+
+### Output
+
+The polygon represented as 2D polytope.
+"""
+function convert(::Type{VPolytope}, P::VPolygon)
```

Comment:
Should we rather use `P::AbstractPolytope`? Then we can use this function for all polytopic types.

Reply:
That's a good idea 👍 

I made them to receive `AbstractPolytope` but it started to mess up with unexpected errors because it was trying to use the new functions, instead of using Julia's default no-op. So we have to add the no-op explicitly.